futurenhs.app/components/_pageTemplates/GroupMemberListingTemplate/index.tsx

```diff
@@ -26,9 +36,23 @@ export const GroupMemberListingTemplate: (props: Props) => JSX.Element = ({
     members,
     contentText,
     pagination,
+    isPublic,
+    csrfToken,
```

Comment:
You should be able to get this via useForm() now

Reply:
I have multiple instances of accept/reject forms for each pending member which are being generated inside the map/memo of the pending members. I can't use useForm inside here as it's not at top level and can't be used inside another hook so I need the csrfToken to come from the page